### ISSUE TITLE
Depth head

### ISSUE THREAD

**Issue by PeggyPeppa** [TuojingAI/ReconDrive#7]
Will the depth head be frozen during training? If it is not frozen, will it cause the depth estimation to collapse or become unstable?

**Comment by primebo1** [comment]
Hi, 

As reported in the paper, our training process is divided into 2 stages. In the first stage, the depth head is trained to ensure the depth is accurate for driving scenes. stage II builds upon this foundation by using the frozen pretrained head.

**Comment by PeggyPeppa** [comment]
May I ask if there are plans to open source the first stage of the training?

**Comment by primebo1** [comment]
> May I ask if there are plans to open source the first stage of the training?

You may be happy to know that it has already been open-sourced : ) see [here](https://github.com/TuojingAI/ReconDrive/blob/main/docs/TRAIN_EVAL.md). Feel free to ask any follow-up questions.

**Comment by PeggyPeppa** [comment]
<img width="1393" height="390" alt="Image" src="https://github.com/user-attachments/assets/02e3f088-c644-45ee-9773-72067e35cf0c" /> Hello, I noticed you only showed the second stage in the training part. Does that mean I only need to modify the corresponding dataloader for training the first stage?

**Comment by PeggyPeppa** [comment]
Also, during the first training stage, the projection loss is not used at all, is it? The model is trained solely under the supervision of rendering loss. Won’t this training paradigm cause the depth to be biased or learned incorrectly, since the network is simultaneously optimizing for Gaussian color and other appearance attributes?
